tests/acceptance/features/bootstrap/OcisConfigContext.php

```diff
@@ -75,6 +75,20 @@ public function theConfigHasBeenSetTo(string $configVariable, string $configValu
 		);
 	}
 
+	/**
+	 * @Given the config :configVariable has been set path to :path
+	 *
+	 * @param string $configVariable
+	 * @param string $path
+	 *
+	 * @return void
+	 * @throws GuzzleException
+	 */
+	public function theConfigHasBeenSetPathTo(string $configVariable, string $path): void {
+		$path = \dirname(__FILE__) . "/../../" . $path;
```

Comment:
maybe you could do this
```suggestion
		$path = UploadHelper::getUploadFilesDir($path);
```